cmd/create_test.go

```diff
@@ -29,6 +29,7 @@ func TestRun(t *testing.T) {
 	mockCC.createConfig.DeployVariables = append(mockCC.createConfig.DeployVariables, mockPortInput, mockAppNameInput)
 	mockCC.createConfig.LanguageVariables = append(mockCC.createConfig.LanguageVariables, mockPortInput)
 	mockCC.templateWriter = &writers.LocalFSWriter{}
+	mockCC.flagVariables = []string{"foo=bar", "foo1=bar1"}
```

Comment:
adding an assertion as well to validate that the flag vars are being correctly captured could boost our coverage here too

Reply:
Added a few tests to validate if correct files are created when using the variable flag. The variables are being validated in the source code, so I've just added an assertion for nil error in the tests .Also, added an integration test for validating the dry-run JSON when the config file is not provided. Let me know what you think.